target/linux/ramips/mt7621/base-files/etc/hotplug.d/firmware/10-ath9k-eeprom

```diff
@@ -22,8 +21,7 @@ case "$FIRMWARE" in
 "ath9k-eeprom-pci-0000:01:00.0.bin")
 	case $board in
 	mikrotik,ltap-2hnd)
-		mac_base="$(cat /sys/firmware/mikrotik/hard_config/mac_base)"
-		caldata_mikrotik_ath9k 0x5000 0x440 $(macaddr_add "$mac_base" 1)
+		caldata_mikrotik_ath9k 0x5000 0x440
```

Comment:
I don't like leaving `macaddr` as 3rd parameter in function `caldata_mikrotik_ath9k`
I think we should be able to directly use `caldata_sysfsload_from_file`, and cut `caldata_mikrotik_ath9k` from this file?
```suggestion
		caldata_sysfsload_from_file "/sys/firmware/mikrotik/hard_config/wlan_data" 0x5000 0x440
```

Reply:
Done. Given that it's only one device, it's probably fine to do this.

Compare to target/linux/ath79/mikrotik/base-files/etc/hotplug.d/firmware/10-ath9k-eeprom